conda-recipe-mkl/meta.yaml

```diff
@@ -30,6 +31,9 @@ requirements:
 test:
   imports:
     - slycot
+  commands:
+    - conda install nose pytest scipy parameterized --yes --quiet
+    - python -c "from slycot import test; test()"
```

Comment:
If you install pytest, why not run pytest?

Reply:
Yup. Will convert to pytest and a "Requires:" section. Thanks for the suggestions.